TheFunctionChangedMyMind/README-en.md

```diff
@@ -0,0 +1,322 @@
+---
+title: The Function that changed my mind
+published: true
+description: How a single function showed me a lot of Kotlin
+tags: kotlin,functions,android,refactor
```

Comment:
Add space after each comma

Reply:
I won't. It's not necessary for the tag system and in some poor designed tags systems it would cause errors. this is just metadata